org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/util/JrtFileSystem.java

```diff
@@ -0,0 +1,322 @@
+package org.eclipse.jdt.internal.compiler.util;
```

Comment:
Is this class moved verbatim or did you change any details? 

If the latter, could you create the change in two steps, where the first is just the verbatim move, so we can see the modifications in the history?